src/core/config/netdata_publisher.h

```diff
@@ -149,7 +151,13 @@
  *
  */
 #ifndef OPENTHREAD_CONFIG_NETDATA_PUBLISHER_MAX_PREFIX_ENTRIES
+
+#if OPENTHREAD_CONFIG_BORDER_ROUTING_ENABLE
+#define OPENTHREAD_CONFIG_NETDATA_PUBLISHER_MAX_PREFIX_ENTRIES \
+    (OPENTHREAD_CONFIG_BORDER_ROUTING_MAX_DISCOVERED_PREFIXES + 4)
```

Comment:
The number of different prefixes (on-mesh or external route) that the publisher will accept and manage the publishing of.

Reply:
okay, so `+ 4` is a cap for manual configurations and prefixes added by the BR itself (e.g. self-generated OMR prefix) ?

Reply:
Yes. `+4` is for OMR and any other prefix (user may add). 
I agree it may be good to use 5 for it (2 for OMR and 3 for other). Will change in new push.

Reply:
Thanks!